prebid/Prebid.js
# Kobler Bid Adapter: add new bid adapter

## Type of change


## Description of change
Added Kobler bidder adapter.

- test parameters for validating bids
```
{
  bidder: 'kobler',
  params: {
        placementId: 'k5H7et3R0'
  }
}
```
In order to see a sample bid from Kobler (without a proper setup), you have to also do the following:
      - Change the [`refererInfo` function](https://github.com/prebid/Prebid.js/blob/master/src/refererDetection.js) to return `'https://www.tv2.no/a/11734615'` as a [`referer`](https://github.com/prebid/Prebid.js/blob/caead3ccccc448e4cd09d074fd9f8833f56fe9b3/src/refererDetection.js#L169). This is necessary because Kobler only bids on recognized articles. 
     - Change the adapter's [`BIDDER_ENDPOINT`](https://github.com/prebid/Prebid.js/blob/master/modules/koblerBidAdapter.js#L8) to `'https://bid-service.dev.essrtb.com/bid/prebid_rtb_call'`. This endpoint belongs to the development server that is set up to always return a bid for the correct `placementId` and page URL combination. 

- contact email of the adapter’s maintainer: bidding-support@kobler.no
- [x] official adapter submission

Link to a PR on the docs repo: https://github.com/prebid/prebid.github.io/pull/2806
